carsus/io/nist/ionization.py

```diff
@@ -154,6 +154,12 @@ def parse_ground_level(row):
             except KeyError:
                 pass
 
+            # To handle cases where the ground level J has not been understood:
+            # Take as assumption J=0
+            # Consider adding a log or warning message???
+            if (np.isnan(lvl["J"])):
```

Comment:
I would try to use:

`if lvl["J"].isna():`

to keep the code most Pandas-friendly.

Reply:
Hello - I tried replacing line 160 from the above with

if (lvl["J"].isna()):

instead in my local copy but then it complains as follows:

AttributeError: 'numpy.float64' object has no attribute 'isna'

I guess this is some Pandas thing (I don't know Pandas well). Is there some syntax thing wrong here?

Reply:
Ok! Then don't worry. I'm approving these changes right now.